py/STREAM/Settings/OtherQuantities.py

```diff
@@ -0,0 +1,6 @@
+from DREAM.Settings.OtherQuantities import OtherQuantities as DREAMOtherQty
+
+class OtherQuantities(DREAMOtherQty):
+    def __init__(self):
+        super().__init__()
+        self.QUANTITIES.append(['stream'])
```

Comment:
why is a separate class with 'otherquesntities' required? Quite vague if all you are doing is adding confinement time capabilities? Could you help me understand (I probably and clearly not) why this is needed?

Reply:
I tried to resolve the warning but did not work. I removed the line 6.

Reply:
It was a suggestion of Mathias